portal/views/teacher/teach_new.py

```diff
@@ -489,7 +490,96 @@ def teacher_move_class_new(request, access_code):
 
 @login_required(login_url=reverse_lazy('login_new'))
 @user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('login_new'))
-def teacher_print_reminder_cards_new(request, access_code):
+def teacher_move_students(request, access_code):
+    klass = get_object_or_404(Class, access_code=access_code)
+
+    # check user is authorised to deal with class
+    if request.user.new_teacher != klass.teacher:
+        raise Http404
+
+    transfer_students = request.POST.get('transfer_students', '[]')
+
+    school = klass.teacher.school
+
+    # get classes in same school
+    classes = school.classes()
```

Comment:
the previous statement gave the list of classes in the school which were not klass. Here, it returns all the classes including klass, if I'm not mistaken.